pkg/tempofb/searchdatamap.go

```diff
@@ -21,7 +21,7 @@ const (
 	maxBufferLen = 900 << 20
 )
 
-type TagCallback func(t string)
+type TagCallback func(t string) (stop bool)
```

Comment:
is there a reason to change this one? i would prefer leaving it alone. we are removing all of this in #2159